packages/dev-env/build.js

```diff
@@ -8,8 +8,9 @@ require('esbuild')
     outdir: 'dist',
     platform: 'node',
     external: [
-      'better-sqlite3',
-      'classic-level',
+      '../plc/node_modules/better-sqlite3/*',
+      '../pds/node_modules/better-sqlite3/*',
+      '../../node_modules/classic-level/*',
```

Comment:
Yupyup, thanks for fixing-up my slop.  The externals config in a monorepo really messes with me!

I think it's already clear to you, but what I'm seeing is that this change seems to alter how `better-sqlite3` is referenced within the build, though it doesn't actually include or exclude anything from the build.

Reply:
heh no worries i've beat my head against this so many times.

i actually think you're change is more the direction we should be going. still poking around the hoisting but seems to be working out